packages/amplication-git-pull-request-service/src/core/diff/visitors/delete-files.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { Difference } from 'dir-compare';
+import { mock } from 'jest-mock-extended';
+import { deleteFilesVisitor } from './delete-files';
+import { missingFolderMock } from './mocks';
+
+describe('Testing the delete files visitor', () => {
+  it('return null when its a new file', () => {
```

Comment:
```suggestion
  it('should return null if diff indicates an addition', () => {
```
To be consistent with the `should` you are doing below.